src/bootstrap/src/core/builder.rs

```diff
@@ -2071,6 +2071,14 @@ impl<'a> Builder<'a> {
             // break when incremental compilation is enabled. So this overrides the "no inlining
             // during incremental builds" heuristic for the standard library.
             rustflags.arg("-Zinline-mir");
+
+            // always pass this after the next `#[cfg(bootstrap)]` update.
```

Comment:
> @onur-ozkan for bootstrap double-checking?

Thanks for the ping, just a nit suggestion from my side:

```suggestion
            // FIXME: always pass this after the next `#[cfg(bootstrap)]` update.
```